.evergreen/publish.sh

```diff
@@ -7,22 +7,28 @@ set -o errexit  # Exit the script with error if any of the commands fail
 ############################################
 #            Main Program                  #
 ############################################
+RELEASE=${RELEASE:false}
 
-echo ${RING_FILE_GPG_BASE64} | base64 -d > ${PROJECT_DIRECTORY}/secring.gpg
-
-trap "rm ${PROJECT_DIRECTORY}/secring.gpg; exit" EXIT HUP
+echo ${RING_FILE_GPG_BASE64} | base64 -d > /tmp/secring.gpg
 
+trap "rm /tmp/secring.gpg; exit" EXIT HUP
 
+git status
 
 export ORG_GRADLE_PROJECT_nexusUsername=${NEXUS_USERNAME}
 export ORG_GRADLE_PROJECT_nexusPassword=${NEXUS_PASSWORD}
 export ORG_GRADLE_PROJECT_signing_keyId=${SIGNING_KEY_ID}
 export ORG_GRADLE_PROJECT_signing_password=${SIGNING_PASSWORD}
-export ORG_GRADLE_PROJECT_signing_secretKeyRingFile=${PROJECT_DIRECTORY}/secring.gpg
+export ORG_GRADLE_PROJECT_signing_secretKeyRingFile=/tmp/secring.gpg
 
 echo "Publishing snapshot with jdk11"
 export JAVA_HOME="/opt/java/jdk11"
 
 ./gradlew -version
-./gradlew publishSnapshots
-./gradlew :bson-scala:publishSnapshots :driver-scala:publishSnapshots -PdefaultScalaVersions=2.11.12,2.12.10
+if [ "$RELEASE" == "true" ]; then
```

Comment:
This can be DRY'd up by assigning the taskname `publishSnapshots` or `publishArchives` to a local variable.

Reply:
Done